perf/istio-install/setup_istio.sh

```diff
@@ -168,11 +168,11 @@ function install_istio() {
   if [[ ! -d "${DIRNAME}/${release}" ]];then
       DN=$(mktemp -d)
       tar -xzf "${outfile}" -C "${DN}" --strip-components 1
-      if [[ -z "${INSTALL_WITH_ISTIOCTL}" ]]; then
+      if ${!INSTALL_WITH_ISTIOCTL:-true}; then
         mv "${DN}/install/kubernetes/helm" "${DIRNAME}/${release}"
-        mv "${DN}/bin/istioctl" "${DIRNAME}/${release}"
       fi
       cp "${DN}/bin/istioctl" "${DIRNAME}"
+      cp "${DN}/manifest" "${DIRNAME}"
```

Comment:
This is a folder, don't we need `-r`? I tihnk its also called `manifests`

Reply:
we removed it in the next line. `rm -rf "${DN}"`

Reply:
i just realized it is called `manifests`, fixed it